deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -2592,7 +2592,13 @@ fun(Conf) ->
     end
 end}.
 
+%% Enable or disable local random exchange
+%%
+%% {exchange_types.local_random.enabled, false},
```

Comment:
```suggestion
```

I think we can just remove this part of the comment. Some of the older options at the top of the file show what is set in advanced.config syntax within the rabbit application, but we can see what the option name is in rabbit within the `mapping` below (the `rabbit.local_random_exchange_enabled` part)